### ISSUE TITLE
intersect lesson01 and lesson02 with a setup-check episode

### ISSUE THREAD

**Issue by psteinb** [carpentries-incubator/deep-learning-intro#100]
depending on the setup of a workshop, it might be worth considering an interlude episode between lesson01 and lesson02 which only looks into the correct installation of all software

this idea is motivated by quite some confusion spurred during the trial teaching where 3/4 learners didn't see a well converging model (performing worse than random draws)

- [x] Add versions to setup instructions
- [x] Add check to setup instruction
- [x] Add reference to instructor notes.

**Comment by bpmweel** [comment]
Perhaps it would make sense to update episode 2's architecture and/or training so it successfully converges instead of the failing it does now.

Although in our teaching recently there was a very important learning moment here. Learners thought based on the training loss that the network would perform well, however on the test set it fails tremendously.
